code/modules/food_and_drinks/recipes/tablecraft/recipes_frozen.dm

```diff
@@ -64,7 +64,7 @@
 		/obj/item/reagent_containers/food/drinks/sillycup = 1,
 		/datum/reagent/consumable/ice = 15
 	)
-	result = /obj/item/reagent_containers/food/snacks/snowcones
+	result = /obj/item/reagent_containers/food/snacks/snowcones/flavourless
```

Comment:
flavourless should just be parent type

Reply:
Tried. straight up didn't work and broke everything else again, If parent isn't specifically set to contain 10 water it for some reason makes the snowcone have 0 contents at all